### PR TITLE
 Add pop up project, and project section

### PR DESCRIPTION
**Pull request by Richie9000**
Add pop-up project, and project section created dynamically using the information stored in that JavaScript object.

### CODE REVIEW

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

mario_ladders

## Good points 👍 
- No Linters error ✔️

## Aspects to improve ♻️

- [ ] When a project is opened, please make sure that project info appears, right now it shows the same info for every project (but I saw you make projects array, good job on that) when I click uber navigation of Facebook 360 it just appears the Topic title with the same info. In other words...
- The projects section should be created dynamically with the information from the projects array
- [ ] Some images are not available on the project
- [ ] The tags have bugs on specific screen sizes
image

- [ ] On mobile size the tags are not available, I kindly invite you to fix it.
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -99,306 +152,98 @@ <h1>
...
-          <div class="barxx">
-            <div class="bar12"></div>
-          </div>
-        </div>
-      </section>
-    </main>
+        </section>
+      </main>
+    </div>
+    <section class="popup"></section>
```
Comment (smcommits) [inline index.html:247]: - [ ] **As the previous reviewer pointed out, the project images are not loaded in the project pop-up. Please fix this**

The images are missing due to a bad path. Please fix the path to images. 

image

Comment (smcommits) [inline index.html:247]: - [ ] **In your pop-up window, the stack list is still missing on mobile view. Please fix this**

image

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @Richie9000, 

### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @Richie9000, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**